docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,6 +1,6 @@
 = RuboCop factory_bot
 
-factory_bot-specific analysis for your projects, as an extension to
+https://www.rubydoc.info/gems/factory_bot[factory_bot]-specific analysis for your projects, as an extension to
```

Comment:
My usual entry point to FB is https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md
The rubydoc page is far less useful.
WDYT?

Reply:
Indeed it is. I updated this PR.